apps/lfx-one/src/app/modules/events/foundation-event-dashboard/foundation-event-dashboard.component.scss

```
@@ -1,2 +1,6 @@
 // Copyright The Linux Foundation and each contributor to LFX.
 // SPDX-License-Identifier: MIT
+
+::ng-deep .foundation-events-stats-card.p-card {
```

Stylelint (17.10.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)